### ISSUE TITLE
v0.2.2 can't be required from node

### ISSUE THREAD

**Issue by yusefnapora** [multiformats/js-multihashing#18]
I'm getting the following with node 6.9.5, npm 3.10.10

```
$ npm install --save multihashing@0.2.2
$ node
> require('multihashing')
Error: Cannot find module 'multihashing'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at repl:1:1
    at sigintHandlersWrap (vm.js:22:35)
    at sigintHandlersWrap (vm.js:96:12)
    at ContextifyScript.Script.runInThisContext (vm.js:21:12)
    at REPLServer.defaultEval (repl.js:346:29)
    at bound (domain.js:280:14)
```

Requiring the index file works:
```
> require('./node_modules/multihashing/src/index')
{ [Function: Multihashing]
 // ... etc
 }
```

`package.json` has `lib/index.js` as the "main" entrypoint, but the `lib` dir doesn't exist in the published module.  `dist` exists, but requiring `dist/index.js` just returns an empty object.

**Comment by dignifiedquire** [comment]
@diasdavid we really should deprecate this module...

@yusefnapora if you can please try and transition to js-multihashing-async which is what we use internally and is much better tested and feature ready.

**Comment by dignifiedquire** [comment]
the actual issue here is that aegir was updated and the package.json wasn't afjusted for the changed build locations

**Comment by yusefnapora** [comment]
Right on; looking through our code, it looks like there's only one place we're actually using it from a synchronous function, and that's not too hard to work around.

**Comment by daviddias** [comment]
@yusefnapora auch, apologies about that, that was because of a old `aegir` version was still being used. Now fixed with 0.2.3. Thanks for the report!

**Comment by yusefnapora** [comment]
No worries, glad I could help.
